### ISSUE TITLE
Fix -Wimplicit-fallthrough warnings in drivers/video/fbdev/sh7760fb.c

### ISSUE THREAD

**Issue by GustavoARSilva** [KSPP/linux#319]
Found after building next-20230620 sh with sh7763rdp_defconfig:
```c
drivers/video/fbdev/sh7760fb.c: In function 'sh7760fb_get_color_info':
drivers/video/fbdev/sh7760fb.c:138:23: warning: this statement may fall through [-Wimplicit-fallthrough=]
  138 |                 lgray = 1;
      |                 ~~~~~~^~~
drivers/video/fbdev/sh7760fb.c:139:9: note: here
  139 |         case LDDFR_4BPP:
      |         ^~~~
drivers/video/fbdev/sh7760fb.c:143:23: warning: this statement may fall through [-Wimplicit-fallthrough=]
  143 |                 lgray = 1;
      |                 ~~~~~~^~~
drivers/video/fbdev/sh7760fb.c:144:9: note: here
  144 |         case LDDFR_8BPP:
      |         ^~~~
```

**Comment by GustavoARSilva** [comment]
Patch: https://lore.kernel.org/linux-hardening/ZJR66f3UY0UFZJ01@work/

**Comment by GustavoARSilva** [comment]
fixed in commit 7947540d1d1e76297869921ed601e2f47fa6f27d
